modules/tensor_mechanics/src/materials/ADSingleVariableReturnMappingSolution.C

```diff
@@ -214,7 +214,7 @@ ADSingleVariableReturnMappingSolution<compute_stage>::internalSolve(
         if (residual_old - _residual != 0.0)
         {
           ADReal alpha = residual_old / (residual_old - _residual);
-          MathUtils::clamp(alpha, 1.0e-2, 1.0);
+          alpha = MathUtils::clamp(alpha, 1.0e-2, 1.0);
```

Comment:
Yea, so if you make the change I suggested above, then you won't need to modify this line